packages/client/src/pages/home/action-group/post-dialog.tsx

```diff
@@ -223,6 +232,9 @@ const PostDialog = ({ open, onClose }: PostDialogProps) => {
             }
           />
         </StyledRowContainer>
+        {errors.tag && (
+          <StyledErrorMessage>{'"tag" cannot be empty'}</StyledErrorMessage>

```

Comment:
I settled for this after 2 hrs of trying to use Joi to produce the same errors as the other two inputs of this dialog. If anyone knows how to do this pls help.. 😭

Reply:
I think we can just manually do it without Joi since our validation is kind of complicated. Essentially we use the input value if the selected value doesn't exist. We can just check this when we submit I think.

Reply:
Nowww I see what you meant by manually lol